bears/general/DuplicateFileBear.py

```diff
@@ -0,0 +1,32 @@
+import itertools
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class DuplicateFileBear(GlobalBear):
+    LANGUAGES = {'All'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Duplication'}
+
+    def run(self):
+        """
+        Checks for Duplicate Files using filecmp module
+        of Standard Library
+        """
+        if not self.file_dict:
+            yield Result(self, 'You did not add any file to compare')
+        else:
+            unique_tuples = [file_tuple for file_tuple in
+                             itertools.combinations(self.file_dict, 2)]
+
+            for file_pair in unique_tuples:
+                if (self.file_dict[file_pair[0]] ==
+                        self.file_dict[file_pair[1]]):
+                    first_file_name = file_pair[0]
+                    second_file_name = file_pair[1]
+                    message = ('File ' + first_file_name + ' is identical'
+                               ' to File ' + second_file_name)
```

Comment:
But its just skipping the Test entirely it seems. Running py.test locally doesn't run test for only that particular Test file and also my test cases cover all the cases ( there can only be three , since its a simple if else block) . Any reason why that happens ?